packages/wallets/src/features/accounts/screens/IDVDocumentUpload/components/IDVDocumentUploadDetails/IDVDocumentUploadDetails.scss

```diff
@@ -0,0 +1,38 @@
+.wallets-idv-document-details {
```

Comment:
v. minor - typically its good habit for the name of class to be the same as component

Reply:
This was done in response to another review (https://github.com/binary-com/deriv-app/pull/11402#discussion_r1392176616) as the classname is too long. In this situation which option would be better?